review_practice/api-server/util/csv-converter.js

```diff
@@ -0,0 +1,33 @@
+const csv = require('csv-parser');
+const fs = require('fs');
+
+const filePath = process.argv[2];
+const fileName = filePath.split('.')[0];
+const targetPath = process.argv[3];
+
+const results = [];
+
+fs.createReadStream(filePath)
+    .pipe(csv())
+    .on('data', data => {
+        for (key in data) {
+            if (data[key] === 'TRUE') data[key] = true;
+            else if (data[key] === 'FALSE') data[key] = false;
+            else if (data[key] === 'null') data[key] = null;
+            else if (!isNaN(data[key])) data[key] = Number(data[key]);
+        }
```

Comment:
```suggestion
        for (const [key, value] of Object.entries(data)) {
            if (value === 'TRUE') data[key] = true;
            else if (value === 'FALSE') data[key] = false;
            else if (value === 'null') data[key] = null;
            else if (!isNaN(value)) data[key] = Number(data[key]);
        }
```
이렇게 `Object.entreis`와 `for-of`를 사용해도 된답니다!

혹은 if문 대신에
```suggestion
        for (const [key, value] of Object.entries(data)) {
            data[key] = {TRUE: true, FASLE: false, null: null}[key];
            if (!isNaN(value)) data[key] = Number(data[key]);
        }
```
이렇게 표현해도 무방합니다!

Reply:
아니면 아예 함수로 for block의 내용을 분리해주면 더 좋을 것 같아요~